lua/hop/init.lua

```diff
@@ -151,6 +162,7 @@ function M.refine_hints(buf_handle, key)
 
     -- JUMP!
     vim.api.nvim_win_set_cursor(0, { h.line + 1, h.col - 1})
+    return h
```

Comment:
What is this used for? :)

Reply:
I need to check whether `h` is nil *in the getchar loop* in the parent function.

Reply:
Sorry, I just realized an explanation is probably in order here: With the original keybinding-based solution, there was no need to check whether we were done. With the `getchar()` loop, we need to know when to stop and return control to Neovim's normal key handling, which is done as soon as `h ~= nil` in the parent function.